tests/ParserTest.php

```diff
@@ -155,7 +162,7 @@ public function colorParsing()
             '#mine {color: red;border-color: #0a64e6;border-color: rgba(10,100,231,.3);outline-color: #222;'
             . 'background-color: #232323;}'
             . "\n"
-            . '#yours {background-color: hsl(220,10%,220%);background-color: hsla(220,10%,220%,.3);}'
+            . '#yours {background-color: hsl(220,10%,220%);background-color: hsla(220,10%,220%,.3);outline-color: #000;}'
```

Comment:
With the change to throw an exception, the result of `render()` should now be as before.

Reply:
If the declaration was already dropped by the previous implementation, then how was it incorrect, exactly?

Reply:
> If the declaration was already dropped by the previous implementation, then how was it incorrect, exactly?

This change adds an invalid declaration `outline-color: #22;` to the test fixture, which wasn't previously present, but which should be dropped.  Prior to the change, a PHP notice "Uninitialized string offset" would occur, but some semblance of the declaration would still be included (I think as if it were `#220000`, if `intval()` on the result of an out-of-bounds string access returns zero).